x-pack/plugins/fleet/server/routes/app/index.ts

```diff
@@ -35,6 +38,36 @@ export const getCheckPermissionsHandler: RequestHandler = async (context, reques
   }
 };
 
+export const regenerateServiceTokenHandler: RequestHandler = async (context, request, response) => {
```

Comment:
Is regenerate really what we want? It means if this API endpoint is called, all existing fleet-servers will stop working. Instead I more thinking of it as "add token".

Reply:
Yes I think we probably want a generate token endpoint here

Reply:
My concern with adding a new token every time is that maybe some of them will get generated and then never used. Is that okay?

Also, since this is a new set of ES APIs, I couldn't find documentation for "retrieve all token names for this service account", which would let me generate incremental token names. Should we just generate the token names based on timestamp? `token-{timestamp}`?